packages/serialization/form/src/formSerializationWriter.ts

```diff
@@ -131,6 +131,17 @@ export class FormSerializationWriter implements SerializationWriter {
 			}
 		}
 	};
+	public writeCollectionOfEnumValue = <T>(key?: string, values?: (T | null | undefined)[]): void => {
+		if (key && values && values.length > 0) {
+			// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+			const rawValues = values.filter((x) => x !== undefined).map((x) => `${x}`);
+			if (rawValues.length > 0) {
```

Comment:
can't we simply pass the rawValues to a writeCollectionOfStringValues?

Reply:
Updated with `writeCollectionOfPrimitiveValues`